subsys/partition_manager/Kconfig

```diff
@@ -49,19 +49,22 @@ if FILE_SYSTEM_LITTLEFS
 partition=LITTLEFS
 partition-size=0x6000
 rsource "Kconfig.template.partition_size"
```

Comment:
In the partition_size. Can you update so that TFM_NONSECURE_STORAGE is not enabled when placed in external flash.

`	default y if PM_PARTITION_SIZE_$(partition) != 0 && !PM_PARTITION_REGION_$(partition)_EXTERNAL`

Seeing as you created a new template for the region perhaps this should have been a template of itself as well, it is not clear that something like that is in the "partition_size" config.

I think maybe either rename to `Kconfig.template.partition_config` and put everything in there, or move the nonsecure stuff to a new template called `Kconfig.template.partition_nonsecure`. I think the first one is the best option.

Reply:
I renamed the file, but kept the region file separate as they are almost never used in pair.